app/org/sagebionetworks/bridge/validators/SurveyAnswerValidator.java

```diff
@@ -157,18 +163,7 @@ private void validateType(Errors errors, TimeBasedConstraints con, String answer
 
     private void validateType(Errors errors, DecimalConstraints con, String answer) {
```

Comment:
validateType(Errors, DecimalConstraints, String) and validateType(Errors, IntegerConstraints, String) differ only by the error message (which we could probably consolidate). Should we delete these and just use validateType(Errors, NumericalConstraints, String) and let Java's dynamic type system take care of this for us? (And move the try-catch.)